### PR TITLE
Add form and set up mocha

### PR DESCRIPTION
**Pull request by hyang77**
Hi everyone! Here are things I did in this pr:

1. I created a feedback form without styling yet.
2. Fixed some errors and warnings detected by linter, and let linter ignore js files in our backstop_data and docs folder.
3. Set up mocha so and now everyone can add the codes you want to test in the test.js file.

### CODE REVIEW

**Review by wxie11 — DISMISSED** [review]
LGTM

File: public/feedback.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      </nav>
+    </header>
+    <main id="content">
+      <h1 id="title">Feedback Form</h1>
+      <form id="form">
+        <label for="subject">Subject:</label>
+        <input type="text" name="subject" id="subject" required/>
+        <label for="email">Your Email:</label>
+        <input type="email" name="email" id="email" required/>
+        <label for="feedback-text">Feedback:/label>
```
Comment (wxie11) [inline public/feedback.html]: < is missing for the end tag

Comment (hyang77) [reply public/feedback.html]: Thanks. I just fixed that.

**Review by wxie11 — DISMISSED** [review]
lgtm

**Review by wxie11 — DISMISSED** [review]
lgtm

**Review by jhuang60 — DISMISSED** [review]
lgtm

**Review by wxie11 — APPROVED** [review]
lgtm

**Review by jhuang60 — APPROVED** [review]
lgtm

**Pull request merged**